tilecloud_chain/server.py

```diff
@@ -466,12 +482,8 @@ def __call__(self):
             for param, value in self.request.params.items():
                 params[param.upper()] = value
 
-        return self.serve(path, params)
-
-    def error(self, code, message=''):
-        raise self.HTTP_EXCEPTIONS[code](message)
+        data, headers = self.server.serve(path, params)
```

Comment:
This this seems weird to me, sometimes we return body to view caller ? not the response object.